gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -127,6 +127,11 @@
             <property name="illegalPkgs" value="junit.framework"/>
             <message key="import.illegal" value="Use JUnit 4-style (org.junit.*) test classes and assertions instead of JUnit 3 (junit.framework.*)."/>
         </module>
+        <module name="IllegalImport"> <!-- Some OpenJDK builds do not include javafx; only relevant for pre-Java 11 b/c javafx is gone completely in Java 11 -->
```

Comment:
This comment isn't necessary if it's also in the error message :-)

Reply:
removed redundant bits